### PR TITLE
Add weapon drops to logs

### PR DESCRIPTION
**Pull request by ntm5**
Adds logging for when CT's drop a weapon. Useful for gunplanting

### CODE REVIEW

File: mod/Jailbreak.Logs/Listeners/LogEntityListeners.cs [hunk 1]
```diff
@@ -15,6 +16,24 @@ public class LogEntityListeners : IPluginBehavior
...
+
+	private void OnDropGun()
+	{
+		WeaponDrop.Hook(hook =>
+		{
+			var player = hook.GetParam<CBasePlayerPawn>(0);
+			var weapon = hook.GetParam<CBasePlayerWeapon>(1);
+
+			if (!player.IsValid) return HookResult.Continue;
+			if (player.TeamNum != (byte)CsTeam.CounterTerrorist) return HookResult.Changed;
```
Comment (MSWS) [inline mod/Jailbreak.Logs/Listeners/LogEntityListeners.cs:32]: Please use player.Team instead of player.TeamNum

**Review by MSWS — CHANGES_REQUESTED** [review]
Could you add a log message for when a prisoner picks up a CT's gun?

File: mod/Jailbreak.Logs/Listeners/LogEntityListeners.cs (hunk 1 above)
Comment (ntm5) [reply mod/Jailbreak.Logs/Listeners/LogEntityListeners.cs]: @MSWS Should be fixed. Needs testing for the weapon pickup

**Review by MSWS — APPROVED** [review]

**Comment by jithatsonei** [comment]
reverting - sigs are now out of date @ntm5

**Comment by Interesting-exe** [comment]
new sig on linux is "55 48 89 E5 41 56 41 55 49 89 D5 41 54 49 89 F4 53 48 89 FB E8 ? ? ? ? 49 89 C6" and it receives CCSPlayer_ItemServices in stead of CBasePlayerPawn

**Pull request merged**